contrib/pkginstaller/package.sh

```diff
@@ -29,16 +33,24 @@ function sign() {
   if [ -f "${entitlements}" ]; then
       opts="--entitlements ${entitlements}"
   fi
-  codesign --deep --sign "${CODESIGN_IDENTITY}" --options runtime --force --timestamp "${opts}" "$1"
+  codesign --deep --sign "${CODESIGN_IDENTITY}" --options runtime --timestamp --force ${opts} "$1"
 }
 
-version=$(cat "${BASEDIR}/VERSION")
-arch=$(cat "${BASEDIR}/ARCH")
+function signQemu() {
+  if [ "${NO_CODESIGN}" -eq "1" ]; then
+    return
+  fi
+  # need to remove any extended attributes, otherwise codesign complains:
+  # qemu-system-aarch64: resource fork, Finder information, or similar detritus not allowed
+  xattr -cr "${qemuBinDir}/qemu-system-${arch}"
+  codesign --sign "${CODESIGN_IDENTITY}" --force --entitlements "${BASEDIR}/hvf.entitlements" "${qemuBinDir}/qemu-system-${arch}"
```

Comment:
(we might need to sign a lot more of the included libraries and binaries inside `qemu/lib, qemu/bin` but want to know before signing everything if only signing the `qemu-system-*` binary passes the apple notarization checks)

Reply:
we do need to sign more, https://github.com/containers/podman/pull/15105#pullrequestreview-1059212224